multimodal/src/autogluon/multimodal/models/fusion/base.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from abc import ABC, abstractclassmethod, abstractmethod
+from typing import Optional
+
+from torch import nn
+
+from ...constants import AUTOMM, LABEL
+
+logger = logging.getLogger(AUTOMM)
+
+
+class BaseMultimodalFusionModel(ABC, nn.Module):
+    """
+    A base class to fuse different models' features (single-modal and multimodal).
+    """
+
+    def __init__(
+        self,
+        prefix: str,
+        models: list,
+        loss_weight: Optional[float] = None,
+    ):
+        super().__init__()
+
+        self.prefix = prefix
+        self.loss_weight = loss_weight
+        self.model = nn.ModuleList(models)
```

Comment:
This line will force the models to be pytorch models.

@yongxinw Will it be problematic if we are going to use SVM as a fusion model?

Reply:
I can move `self.model` out of the base class, or it can also be overwritten in the child class.

Reply:
`self.model` is for the single models. The fusion part is `self._fusion_xxx`.